data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/FutureCallback.java

```diff
@@ -0,0 +1,47 @@
+///*
+// *  Copyright (c) 2020, 2021 Microsoft Corporation
+// *
+// *  This program and the accompanying materials are made available under the
+// *  terms of the Apache License, Version 2.0 which is available at
+// *  https://www.apache.org/licenses/LICENSE-2.0
+// *
+// *  SPDX-License-Identifier: Apache-2.0
+// *
+// *  Contributors:
+// *       Microsoft Corporation - initial API and implementation
+// *
+// */
+//
+//package org.eclipse.dataspaceconnector.ids.api.multipart.dispatcher;
+//
+//import okhttp3.Call;
+//import okhttp3.Callback;
+//import okhttp3.Response;
+//import org.jetbrains.annotations.NotNull;
+//
+//import java.io.IOException;
+//import java.util.concurrent.CompletableFuture;
+//import java.util.function.Function;
+//
+///**
+// * Shim from OK HTTP to the future returned by the ids dispatcher.
+// */
+//class FutureCallback<T> implements Callback {
```

Comment:
What's the point to checking in classes that entirely commented out?

Reply:
Draft PR :) still need to merge two more branches (one with my next commit) until this code becomes somewhat useable